solutions/Makefile

```diff
@@ -37,8 +36,15 @@ DIRS += redis
 
 build: $(DIRS)
 
+# If BUILD_RUN_CLEAN is set, then build/run/clean each test
```

Comment:
Instead of having an if/else statement if ​BUILD_RUN_CLEAN​ is set, could you just define the ​tests_clean​ stage as:

```
tests_clean: tests
@ $(MAKE) -s clean
```